packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -167,21 +162,11 @@ async function handleGrant(config: ReleaseGoldConfig, currGrant: number) {
       from: fromAddress,
     },
   ])
-  const initializerAbiRG = checkAndReturnInitializationAbi(ReleaseGold, 'ReleaseGold')
-  const transferImplOwnershipAbiRG = checkAndReturnTransferOwnershipAbi(ReleaseGold, 'ReleaseGold')
   console.info('  Deploying ReleaseGoldProxy...')
   const releaseGoldProxy = await retryTx(ReleaseGoldProxy.new, [{ from: fromAddress }])
   console.info('  Deploying ReleaseGold...')
   const releaseGoldInstance = await retryTx(ReleaseGold.new, [{ from: fromAddress }])
```

Comment:
Don't we need to provide a `false` argument for the `ReleaseGold` deployment somewhere in this file? Also more generally, do we have an integration test for this script that would capture such a regression?

Reply:
Great catch! I think this should get flagged by the protocol-test-release check so I'm holding off on fixing it for now to make sure.

Reply:
bummer doesn't look like it was caught. I'll make this PR a draft again and try to figure out why

Reply:
Okay @martin the last few commits address this concern. I've confirmed that the RG deployment script is now run in CI. Unbelievable catch! This would've been a headache in the next deployment.